feat: improve excel command (DEV-955) (#228)
dasch-swiss/dsp-tools: docs/dsp-tools-create.md

Keywords are represented as an array of strings and are used to describe and/or tag the project.

### Lists

(optional)

`"lists": [<list-definition>,<list-definition>,...]`

Lists can be used to provide controlled vocabularies. They can be flat or hierarchical. One advantage of the use of
hierarchical lists is that it allows a user to sub-categorize objects. This helps in the formulation of specific search
requests. If there is a list node "Vocal music" and sub-nodes "Song" and "Opera", a search for "Vocal Music" would
return objects classified as "Song" and "Opera". But a search for "Song" would only return objects classified as "Song".
 
The "lists" section is an array of list definitions. A list definition has one root node whose name is used to identify 

> the list. The children of the root node are the actual list nodes. If the list is hierarchical, the list nodes can have

Reviewer: ```suggestion
the list. The children of the root node are the list nodes. If the list is hierarchical, the list nodes can have
```

> children on their own, and these children can again have children.

Reviewer: ```suggestion
children, and these children can again have children etc.
```

> Once a list is defined in a project, resources can access it via a list-property, e.g. a property with 

Reviewer: ```suggestion
When a list is defined for a project, its values can be referenced in resources within a list property, e.g. a property with 
```

[object "ListValue"](./dsp-tools-create-ontologies.md#listvalue).

A node of a list may have the following elements:

- `name` (mandatory): Name of the node. Has to be unique within the entire "lists" section.
- `labels` (mandatory): Label with language tags in the form `{"<lang>": "<label>", "<lang>": "<label>", ... }`. 
  At least one language needs to be specified. Currently, "de", "en", "fr", "it", and "rm" are supported.

> - `comments` (mandatory for root node, optional for normal nodes): Comment with language tags in the form 

Reviewer: ```suggestion
- `comments` (mandatory for root node, optional for all other nodes): Comment with language tags in the form 
```

                                    "labels": {
                                        "de": "Reptilien",
                                        "en": "Reptiles"
                                    }
                                }
                            ]
                        },
                        {
                            "name": "plantes",
                            "labels": {
                                "de": "Pflanzen",
                                "en": "Plantes"
                            }
                        },
                        {
                            "name": "weather",
                            "labels": {
                                "de": "Wetter",
                                "en": "Weather"
                            }
                        },
                        {
                            "name": "physics",
                            "labels": {
                                "de": "Physik",
                                "en": "Physics"
                            }
                        }
                    ]
                }
            ]
        }
    ]
}
```

#### Lists from Excel

> Instead of being fully defined in JSON, a list can be imported from one or several Excel files. In this case, the 

Reviewer: ```suggestion
Instead of being described in JSON, a list can be imported from one or several Excel files. In this case, the 
```

```json
{
    "lists": [
        {
            "name": "colors",
            "labels": {
                "de": "Farben",
                "en": "Colors"
            },
            "comments": {
                "de": "Eine Liste mit einigen Farben",
                "en": "A list with some colors"
            },
            "nodes": {
                "folder": "path-to-folder"
            }
        },
        {
            "name": "category",
            "labels": {
                "de": "Kategorie",
                "en": "category"
            },
            "comments": {
                "de": "Eine Liste mit Kategorien",
                "en": "A list with categories"
            },
            "nodes": [
                ...
            ]
        }
    ]
}
```

To do so, it would be necessary to place the following two files into the folder "path-to-folder":
![Colors_en](./assets/images/img-list-english-colors.png)
![Farben_de](./assets/images/img-list-german-colors.png)

The expected format of the Excel files is documented 
[here](./dsp-tools-excel.md#create-the-lists-section-of-a-json-project-file-from-excel-files). The only difference to 

> the explanations there is that the Excel column A is not interpreted as list name, but as node name of the first 

Reviewer: ```suggestion
the explanations there is that column A of the Excel worksheet is not interpreted as list name, but as node name of the first 
```
